nucypher/characters/lawful.py

```diff
@@ -117,7 +106,6 @@
 
 class Alice(Character, BlockchainPolicyAuthor):
     banner = ALICE_BANNER
```

Comment:
Can these banners (also Bob's and Enrico's) also be removed, or are they also used somewhere else?

Reply:
The remaining banners here are sent to standard output when initializing a character, which is both cute and a good landmark in logs.